### DISCUSSION TITLE
voice screen sharing

### DISCUSSION THREAD

**Discussion by ShuKiTaRy** [stoatchat/discussions#528] [Feature Suggestions]
Add voice screen sharing please I find it cool if you add it, i am french !

**Comment by sussycatgirl** [comment]
Screen sharing has been proposed numerous times and is certainly planned. As for you being french, my condolences but I'm afraid there's nothing we can do about that

**Reply by Eckoa** [reply]
> As for you being french, my condolences but I'm afraid there's nothing we can do about that


glad i searched for status on screen sharing because this gave me a good chuckle
